near-sdk/src/lib.rs

```diff
@@ -99,10 +101,321 @@
 #[cfg(test)]
 extern crate quickcheck;
 
-pub use near_sdk_macros::{
-    ext_contract, near, near_bindgen, BorshStorageKey, EventMetadata, FunctionError, NearSchema,
-    PanicOnDefault,
-};
+/// This attribute macro is used on a struct and its implementations
+/// to generate the necessary code to expose `pub` methods from the contract as well
+/// as generating the glue code to be a valid NEAR contract.
+///
```

Comment:
it should mention here with a reference that it's syntactic sugar for `near_bindgen` macro, 
and expands to `near_bindgen` macro invocation